Makefile

```
@@ -8,6 +8,8 @@ PULL_PARAM=$(shell if [ "${CONTAINER_COMMAND}" = "podman" ];then echo "--pull-al
 
 ROOT_DIR = $(shell dirname $(realpath $(firstword $(MAKEFILE_LIST))))
 PYTHONPATH=$(shell scripts/utils.sh get_pythonpath ${ROOT_DIR})
+PATH=$(shell scripts/utils.sh get_path)
```

```suggestion
PATH := "${PATH}:/usr/local/bin"
```

``./create_full_environment.sh`` is either way being called in the same shell, and no reason for it to pollute outside shell scope